backend/mora/service/manager.py

```diff
@@ -13,20 +13,145 @@
 
 """
 import uuid
+import flask
 
 from . import address
 from . import handlers
+from . import orgunit
+from . import facet
+from . import employee
 from .. import common
 from .. import lora
 from .. import mapping
 from .. import util
 from .. import validator
 
+SEARCH_FIELDS = {
+    'e': 'tilknyttedebrugere',
+    'ou': 'tilknyttedeenheder'
+}
 
-class ManagerRequestHandler(handlers.OrgFunkRequestHandler):
+
+class ManagerRequestHandler(
+    handlers.OrgFunkRequestHandler,
+    handlers.ReadingRequestHandler
+):
+
+    __slots__ = ()
     role_type = 'manager'
     function_key = mapping.MANAGER_KEY
 
+    @classmethod
+    def has(cls, scope, objid):
+        pass
```

Comment:
This should be implemented; I believe the phone book uses it. (The address handler may also lack one; I'd consider that a bug rather than a feature.)